doc/cephfs/scrub.rst

```diff
@@ -150,6 +150,8 @@ These above named MDS damages can be repaired by using the following command::
 If scrub is able to repair the damage, the corresponding entry is automatically
 removed from the damage table.
 
+Note: A scrub invoked with the ``repair`` option can identify an damaged hard link but not repair it.
```

Comment:
The article should be "a damaged" instead of "an damaged".

I'd also suggest to use an admonition, something like

```suggestion
.. note:: A scrub invoked with the ``repair`` option can identify a damaged hard link but not repair it.
```